### PR TITLE
Added New Emojicloud (V1)

### PR DESCRIPTION
**Pull request by dakshAg**
New Fancy Emojicloud

- Better looking Unicode Emojis
- Custom Emojis (Animated too!)
- Alex's version is still there at `/wordcloud`
- Use `/emojicloud` to activate this 

Lightly Tested. I expect it to be imperfect but will get there in a few iterations and with feedback. Hopefully. 🤞

output

### CODE REVIEW

**Comment by adamnwerbik** [comment]
I am hyped beyond belief.

**Comment by olliequ** [comment]
I am belief beyond hyped.

**Comment by acmahaja** [comment]
I am

File: requirements.txt [hunk 1]
```diff
@@ -37,4 +37,6 @@ urllib3==1.26.9
 uvloop==0.16.0; sys_platform != "win32"
 yarl==1.7.2
 humanize==4.3.0
-wordcloud==1.8.2.2
\ No newline at end of file
+wordcloud==1.8.2.2
+uvloop~=0.16.0
```
Comment (flukiluke) [inline requirements.txt]: Duplicate of above (uvloop)

File: emoji_cache.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import os
+
+import lightbulb
+from PIL import Image
+
+import db
+
+
+def get_f_name(emoji):
+    return f"cache/{emoji.filename}"
```
Comment (flukiluke) [inline emoji_cache.py]: Please store all generated files in the `assets` directory

File: extensions/emojicloud.py [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+        for j in range(len(listofimages)):
+            p = listofimages[j]
+            ti = all_thumbnails[j]
+            if ti[1] == "custom":
+                ti[0].seek(fi % ti[0].n_frames)
+            resized_img = ti[0].resize((p[2], p[2]))
+            new_im.paste(resized_img, (p[0], p[1]), resized_img.convert('RGBA'))
+            frames.append(new_im)
+
+    frames[0].save('output.gif', save_all=True, append_images=frames)
```
Comment (flukiluke) [inline extensions/emojicloud.py]: Move to `assets` (or `/tmp`)

**Review by flukiluke — CHANGES_REQUESTED** [review]

**Comment by dakshAg** [comment]
Resolved Suggestions by Luke

**Pull request merged**